.gitignore

```diff
@@ -36,3 +36,4 @@ k8sdumps
 
 electron/env.ts
 
+get_helm.sh
```

Comment:
What is `get_helm.sh` ?

Reply:
Ir automatically generated while running build. I will remove it.